scripts/build-with-docker.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash -ex
```

Comment:
We don't really need this here as we can just use the regular `snapcraft` as is to build the snap. We're not running this in the Anbox Cloud CI where we cannot use the snapcraft snap. So lets drop this.

Reply:
Okay.